vochain/scrutinizer/vote.go

```diff
@@ -121,11 +133,17 @@ func (s *Scrutinizer) GetEnvelopes(processId []byte) ([]*models.VoteEnvelope, []
 			if envelope == nil {
 				return fmt.Errorf("transaction is not an Envelope")
 			}
-			envelopes = append(envelopes, envelope)
-			weights = append(weights, txRef.Weight)
+			envelope.Nullifier = txRef.Nullifier
+			envelopes = append(envelopes, &types.EnvelopeMetadata{
```

Comment:
Cannot we reuse EnvelopePackage type here?

Reply:
for the `getEnvelope` call we want `EnvelopePackage` because it contains the entire vote package itself (and other detailed information). for `getEnvelopeList`, we want to serve as little information as possible for each envelope, which means not sending the EncryptionKeyIndexes, Nonce, Signature, VotePackage, and Weight. 
I guess we could use a `EnvelopePackage` and simply leave some of the fields blank, I dislike this because I find it easy to mistakenly use a field that's never going to be set in this context. But I could easily make this change if it's better :)

Reply:
There is another option which is keep EnvelopeMetadata{} but use it also as a field of EnvelopePackage{} so we are unifying and not repeating fields on two different structs.